### PR TITLE
Fix admin posting card event date range 

### LINKED ISSUE #616: Admin posting card event should not have date range
Should just be "Saturday, Aug 20, 2022" without the range

similar to what we have for volunteer posting cards:

### PR DESCRIPTION
**Pull request by briantu**
## Ticket link

Closes #616

## Implementation description
* Fixed admin posting card to only show single date if posting is an event
* ~~Fixed time range in volunteer posting card to show correct time (was offset by 4 hours the posting shift times)~~

## Steps to test
1. Go to `http://localhost:3000/admin`
2. Create event posting
3. Check posting card date is not shown as range anymore
4. Check volunteer posting card time range is correct

## What should reviewers focus on?
* Code correctness

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/utils/DateTimeUtils.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { FilterType } from "../types/DateFilterTypes";
  * @returns corresponding time string string in format hh:mm:(a/p)m
  */
 export const formatTimeHourMinutes = (date: Date): string => {
-  return moment.utc(date).format("h:mma");
+  return moment(date).format("h:mma");
```
Comment (briantu) [inline frontend/src/utils/DateTimeUtils.ts:11]: Not sure if this is the correct fix because iirc I thought we are using UTC?

Comment (MatoPlus) [reply frontend/src/utils/DateTimeUtils.ts:11]: yea im reworking this as well, you can leave this to me if you want: https://github.com/uwblueprint/sistering/pull/633

**Review by MatoPlus — COMMENTED** [review]
seems good to me. Will approve in a bit after some tests

File: frontend/src/utils/DateTimeUtils.ts (hunk 1 above)
Comment (briantu) [reply frontend/src/utils/DateTimeUtils.ts:11]: Sure, will leave it to you

Comment (MatoPlus) [reply frontend/src/utils/DateTimeUtils.ts:11]: Awesome, please revert this. seems like a few usage break due to the change 😢. gotta be more granular with the change

File: frontend/src/components/pages/admin/AdminHomepage.tsx [hunk 2]
```diff
@@ -274,6 +275,14 @@ const AdminHomepage = (): React.ReactElement => {
                       autoClosingDate={posting.autoClosingDate}
                       branchName={posting.branch.name}
                       numVolunteers={posting.numVolunteers}
+                      type={
```
Comment (MatoPlus) [inline frontend/src/components/pages/admin/AdminHomepage.tsx:278]: Why do we need to drill this? Would it be okay to move this into `getPostingDateRange` logic?

**Review by MatoPlus — CHANGES_REQUESTED** [review]
Made 2 suggestions, LGTM otherwise

File: frontend/src/components/pages/admin/AdminHomepage.tsx (hunk 2 above)
Comment (briantu) [reply frontend/src/components/pages/admin/AdminHomepage.tsx:278]: Sure, I can do that

**Review by MatoPlus — APPROVED** [review]
LGTM!

**Pull request merged**